### PR TITLE
1279 notes to sefa workbook

### PR DESCRIPTION
**Pull request by sambodeme**
## Description
- Added Notes for SEFA Workbook

### CODE REVIEW

**Review by JeanMarie-TTS — COMMENTED** [review]
@sambodeme There can be any number of notes in the notes_to_sefa. Please confirm that there is a sequence number field in each entry. I'm not sure how to confirm this by looking at the changes in the PR.

**Comment by jadudm** [comment]
We don't currently have an explicit sequence number in the sheets. Do we need an auto-generated seq_number? If so, does it cross-validate against another sheet?

**Comment by JeanMarie-TTS** [comment]
> We don't currently have an explicit sequence number in the sheets. Do we need an auto-generated seq_number? If so, does it cross-validate against another sheet?

There is no cross validation against anything else. However, we do need a sequence number as there is nothing else to distinguish one note from another. I'd rather not have ETL generate a sequence number.

**Comment by jadudm** [comment]
Got it.

@sambodeme , do you think we can add a hidden seq_number field that auto-populates? Or, does this take us into the space of other questions that we haven't had a chance to touch base on? 

Happy to do a call to figure out what is the best/most consistent approach here.

**Comment by sambodeme** [comment]
> @sambodeme There can be any number of notes in the notes_to_sefa. Please confirm that there is a sequence number field in each entry. I'm not sure how to confirm this by looking at the changes in the PR.
@JeanMarie-TTS, @jadudm  I am including it now. I started to include this seq number by default for xlsx files that involve multiple entries but I was advised that it may not always be necessary, so I backed off that plan. 
Please let me know the xlsx files you need a sequence number for, and I will make sure to address them in a separate PR (if necessary).

**Comment by jadudm** [comment]
How about we add any seq_number additions here:

https://github.com/GSA-TTS/FAC/issues/1471

which we know we want to do, but it puts it on an epic that's about touching all of the workbooks in a consistent way/final way before going liev.

**Comment by sambodeme** [comment]
> How about we add any seq_number additions here:
> 
> #1471
> 
> which we know we want to do, but it puts it on an epic that's about touching all of the workbooks in a consistent way/final way before going liev.

That was my intention, but it seems that this won't be useful if the only remaining Excel file is this one, as I have already added the sequence number to it. However, if there are more files that require a sequence number, I will definitely open a ticket and add it to the queue. Previously, I created a ticket for this, but JeanMarie mentioned that the required sequence numbers are already addressed through another ticket, so I closed mine. Therefore, I wanted to confirm if there is still a need for it before creating another ticket.

File: backend/schemas/source/excel/templates/notes-to-sefa-template.jsonnet [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    Sheets.open_range {
+      width: 56,
+      help: Help.plain_text,
+    },
+    SV.NoValidation,
+    'Note content',
+    'note_content',
+  ],
+  [
+    Sheets.open_range {
```
Comment (sambodeme) [inline backend/schemas/source/excel/templates/notes-to-sefa-template.jsonnet:71]: This should be read-only and hidden (if necessary). The logic to enforce these two requirements is currently sitting in #1175. Depending on which ever gets merged first, I will update the next one accordingly.

**Review by jadudm — APPROVED** [review]
It is the Notes to SEFA form. We have additional updates coming as part of #1175 as well the final walkthrough of the notebooks. LGTM, and thank you!

**Pull request merged**